src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/TemplatesCatalogDrawerPanel.tsx

```diff
@@ -75,11 +89,45 @@ export const TemplatesCatalogDrawerPanel: React.FC<TemplatesCatalogDrawerPanelPr
     const interfaces = getTemplateInterfaces(template);
     const disks = getTemplateDisks(template);
     const isDefaultTemplate = isDefaultVariantTemplate(template);
-    const { memory, cpuCount } = getTemplateFlavorData(template);
     const hostDevicesCount = getHostDevices(vmObject)?.length || 0;
     const gpusCount = getGPUDevices(vmObject)?.length || 0;
     const hardwareDevicesCount = hostDevicesCount + gpusCount;
 
+    const [updatedVMObject, setUpdatedVMObject] = useState<V1VirtualMachine>(undefined);
+
+    useEffect(() => {
+      k8sCreate<V1Template>({
+        model: ProcessedTemplatesModel,
+        data: setValueForRequiredParams(template),
```

Comment:
can you please explain to me why is setValueForRequiredParams needed?
we should get the PARAM values when processing the template, is that not working?

Reply:
I've experienced some scenarios, when for some templates, processing the template failed due to not having the param values already there. Not sure why this happens and only for some templates.